src/plugins/intel_cpu/src/mkldnn_node.h

```diff
@@ -582,6 +583,17 @@ class MKLDNNNode {
     */
     std::pair<std::vector<float>, std::vector<float>> getScalesAndShifts(const MKLDNNNode *parentNode) const;
 
+    /**
+     * @brief Appends new item into ops list with the information on how the node should be executed as post operation.
+     * Seed node should call this routine and pass its post operations list as parameter.
+     * @param ops List of fused post operations
+     */
+    virtual void appendPostOps(mkldnn::post_ops& ops, const VectorDims& postOpDims);
```

Comment:
Please take a look at https://github.com/maxnick/openvino/blob/396cf7b85bf75c7e08ad7a51697fa6e786e8507e/src/plugins/intel_cpu/src/nodes/mkldnn_eltwise_node.cpp#L1926-L1928
There we call `appendPostOps` using an instance of the base class, but we can access protected members only though `this` pointer since this is a part of the current instance. So to do this call in other parts of the plugin code we usually use dynamic casting to a specific type that looks rather weird. May be it had sense earlier but now I can not see any arguments against making this method public. Moreover, this method usually implemented as public in derived classes. The only reason it is in the `protected` section, is that is not implemented for the base type, I mean it throws. So m.b. that is the reason. But, following this logic we can simply remove this method from the basic node at all, and implement it only in specific node type, since we use type casting anyway.

Reply:
`appendBinaryPostOps` was also moved to the `public` section.